README.md

```diff
@@ -232,7 +232,8 @@ If you were previously using Shopify's [activeresource fork](https://github.com/
 Download the source code and run:
 
 ```bash
-rake install
+bundle install --jobs=3 --retry=3
```

Comment:
These arguments are only needed when running in the CI environment to make sure that the build doesn't die if there's an issue reaching Rubygems.

`--jobs=3` is used for parallelization and mileage might vary on different workstations. Best to omit it as to not cause confusion.